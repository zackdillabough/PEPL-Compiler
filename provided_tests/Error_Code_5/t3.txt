POLY T(X, Y) = X + Y;
POLY S(A, B, C, D, E, F, G, H, I, J, K, L, M, N, O, P, Q, R, T, U, V, W, X, Y, Z) = R^2 + Y^2 + U;
POLY Y = x + 5555;
START
INPUT A;
INPUT B;
INPUT C;

T
(
P, B
);

S(A, B, C, D, E, F, G, 
H, I, J, K, L, M, N, 
O, P, Q, R, T, U, 
V, W, X, Y, Z);

T(1, 2);

INPUT B;
INPUT C;

Y(Q);
Y(P);

T(B, 
M);

1 2 3 4 5
