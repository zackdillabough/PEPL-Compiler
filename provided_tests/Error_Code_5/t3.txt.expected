Error Code 5: 11 14 14 14 14 15 15 15 15 15 15 15 16 16 16 16 16 16 17 17 17 17 17 24 25 28 